apps/oxlint/src/lint.rs

```diff
@@ -115,6 +108,18 @@ impl Runner for LintRunner {
             Oxlintrc::default()
         };
 
+        let cwd = std::env::current_dir().unwrap();
+
+        let paths = Walk::new(
+            &paths,
+            &ignore_options,
+            &oxlintrc.ignores.iter().map(|value| cwd.join(value)).collect::<Vec<_>>(),
```

Comment:
This joins with the current working directory, but maybe it should be based on the config file path?

Reply:
Using the config file path is also problematic because you might store all your config in a configs folder, so then all your globs would need to traverse up to get out of it.

Maybe the current working directory is the easiest to explain right now. When thinking about nested config files I feel like it becomes less clear though.

Reply:
Usually use case:
1) simple project: eslint config in root folder
2) Monorepo setup: each folder can contain eslint config extended from base root config file and then ovverride or add own rules and etc

Reply:
I recently added a `path` property to the `Oxlintrc` struct which stores a path to the configuration file. Feel free to use this, as I'm sure we'll probably need to do it anyway once we support nested configurations. It at least gets us one degree of abstraction away from "current working directory" as the only notable directory.